tests/utils.py

```diff
@@ -39,9 +44,25 @@ def get_mock_file_manager():
     return file_manager
 
 
+def get_fake_cpu(num_packages, num_cpus):
```

Comment:
I think you should be able to reuse this function: https://github.com/Netflix-Skunkworks/titus-isolate/blob/master/titus_isolate/model/processor/config.py#L26-L29

Reply:
thanks, done